jobs/JOBSPROC_MODS

```diff
@@ -169,15 +169,17 @@ cp $UTILROOT/ush/setpdy.sh .; chmod +x setpdy.sh
 ##############################
 # Run setpdy and initialize PDY variables
 ##############################
-setpdy.sh
+setpdy.sh 9
```

Comment:
Do we need this "9"? I think it's leftover from Xiaoxue's testing.

Reply:
Yes, it is needed. `setpdy.sh` with no parameters will default to setting PDYm and PDYp back (m for minus) 7 days and forward (p for plus) 7 days.  Passing in the parameter will override the default of 7 and calculate backward and forward 9 days. The variable PDYm9 is used 2 lines lower to help set the new MODSDIR location.